pkgs/test/test/runner/pause_after_load_test.dart

```diff
@@ -30,8 +30,8 @@ void main() {
 import 'package:test/test.dart';
 
 void main() {
-  print('loaded test 2!');
```

Comment:
seems a weird delta...

Reply:
Yeah I noticed that. Just ran dartfmt though.